create-environment/action.yml

```diff
@@ -11,6 +11,18 @@ inputs:
     description: 'Power Platform password. Do NOT checkin password, instead create a secret and reference it here with: see: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow'
```

Comment:
update username description as well for both create-env and whoami with the similar text as you have for appID